src/Streamly/Internal/Data/Stream/IsStream/Reduce.hs

```diff
@@ -235,10 +106,40 @@ module Streamly.Internal.Data.Stream.IsStream.Nesting
     -- , slidingSessionBuffer
     -}
 
-    -- * Transform (Nested Containers)
-    -- Nested splitting
+    -- ** Reduce By Parsers
```

Comment:
Reduce by fold had a level 1 heading `-- * Reduce by folds`

Reply:
ok, will change it.